### ISSUE TITLE
Unable to log into Dissenter extension on Brave Using Gab Log in

### ISSUE THREAD

**Issue by BrutalRage** [gab-ai-inc/gab-dissenter-extension#92]
Unable to log into Dissenter extension on Brave Using Gab Log in.  Keep getting the same message white is attached.
![2019-07-05 - Message after trying to log in](https://user-images.githubusercontent.com/52712192/60903739-2508b100-a22f-11e9-8c79-044e35dd9752.JPG)

**Comment by BrutalRage** [comment]
I am using the plugin for Brave

**Comment by justagoodperson** [comment]
+1 was able to reproduce, this is a backend issue as well nothing we can do about it at the moment

**Comment by BrutalRage** [comment]
> +1 was able to reproduce, this is a backend issue as well nothing we can do about it at the moment

Do we know what is causing it?

**Comment by justagoodperson** [comment]
Probably the backend is only allowing browsers that pass certain headers to complete authentication. Brave/others are probably detected as unsupported browsers. My guess is if you tried to do the same with Chromium you would get the same error. Have to have a dev go in and whitelist those headers being passed (just a hunch but I think it's right)

**Comment by NathanHannon** [comment]
This is happening on all browsers on Windows and Linux (and presumably macOS). nothing we an do except wait

**Comment by justagoodperson** [comment]
Has Torba commented on it yet? Maybe if enough people alert him on Gab he'll give an update.

**Comment by YoRHa-A5** [comment]
Same for me even on Dissenter Browser on Manjaro Linux.

**Comment by YoRHa-A5** [comment]
Duplicate of #89

**Comment by Innomen** [comment]
> Probably the backend is only allowing browsers that pass certain headers to complete authentication. Brave/others are probably detected as unsupported browsers. My guess is if you tried to do the same with Chromium you would get the same error. Have to have a dev go in and whitelist those headers being passed (just a hunch but I think it's right)

Can confirm. /Eloston/ungoogled-chromium (64)

**Comment by rugabunda** [comment]
> 
> 
> > Probably the backend is only allowing browsers that pass certain headers to complete authentication. Brave/others are probably detected as unsupported browsers. My guess is if you tried to do the same with Chromium you would get the same error. Have to have a dev go in and whitelist those headers being passed (just a hunch but I think it's right)
> 
> Can confirm. /Eloston/ungoogled-chromium (64)

This happens in dissenter browser as well.

**Comment by rugabunda** [comment]
> 
> 
> Probably the backend is only allowing browsers that pass certain headers to complete authentication. Brave/others are probably detected as unsupported browsers. My guess is if you tried to do the same with Chromium you would get the same error. Have to have a dev go in and whitelist those headers being passed (just a hunch but I think it's right)

This happens in dissenter browser in windows, as well as a standard Firefox browser installation in linux.

**Comment by Innomen** [comment]
Working for me as of 2019-07-11 0131 PM

**Comment by BrutalRage** [comment]
I confirm it is working on 2019-07-11 on Brave.

**Comment by YoRHa-A5** [comment]
I get `Security verification failed. Are you blocking cookies?` even with Shields down and Site Bleacher set to whitelist. 🤔

_Edit: Temporarily deactivating Privacy Possum fixed it._

**Comment by grahamperrin** [comment]
> … all browsers on Windows and Linux (and presumably macOS). …

### CSP

Is this a variant of the same issue? 

![image](https://user-images.githubusercontent.com/192271/61184018-6ee30400-a640-11e9-9ee6-3ccd221a02d9.png)

>> Blocked by Content Security Policy

– https://redd.it/cd2av4

**Comment by rugabunda** [comment]
Is working for me now as well, on all platforms and browsers.
